README.md

```diff
@@ -196,59 +221,31 @@ If you want to use a custom database, you will need to create a Sourmash sketch
 If you have a single FASTA file with _one genome_ per record:
 
 ```bash
-sourmash sketch dna -f -p k=31,scaled=1000,abund --singleton <your multi-FASTA file> -o training_database.sig.zip
+yacht sketch ref --infile <path to your multi-FASTA file> --kmer 31 --scaled 1000 --outfile training_database.sig.zip
```

Comment:
Let's note somewhere that this is equivalent to that deleted `sourmash` command. I don't think it's worth hiding that this is a wrapper

Reply:
I have added a comment to illustrate the equivalent `sourmash` command.